### PR TITLE
Retry the query about interfaces to the qemu-agent

### PR DESCRIPTION
**Pull request by inercia**

### CODE REVIEW

File: libvirt/qemu_agent.go [hunk 1]
```diff
@@ -24,75 +26,98 @@ type QemuAgentInterfaceIpAddress struct {
 	Prefix  uint   `json:"prefix"`
 }
 
+type libvirtIfacesLst []libvirt.DomainInterface
```
Comment (monstermunchkin) [inline libvirt/qemu_agent.go]: There's no real benefit in doing this. Simply writing `[]libvirt.DomainInterface` makes it more understandable IMHO.

File: libvirt/qemu_agent.go [hunk 2]
```diff
@@ -24,75 +26,98 @@ type QemuAgentInterfaceIpAddress struct {
...
-		"{\"execute\":\"guest-network-get-interfaces\"}",
-		libvirt.DOMAIN_QEMU_AGENT_COMMAND_DEFAULT,
-		0)
-	if err != nil {
-		return interfaces
-	}
+func getDomainInterfacesViaQemuAgent(domain LibVirtDomain, wait4ipv4 bool) libvirtIfacesLst {
 
-	log.Printf("[DEBUG] qemu-agent response: %s", result)
+	qemuAgentInterfacesRefreshFunc := func() resource.StateRefreshFunc {
```
Comment (monstermunchkin) [inline libvirt/qemu_agent.go]: Since this function is quite long, we should declare it outside of `getDomainInterfacesViaQemuAgent`.

**Comment by inercia** [comment]
[terraform-libvirt qemu-agent PR](https://trello.com/c/PpVTYGGE/151-terraform-libvirt-qemu-agent-pr)

**Comment by inercia** [comment]
This is probably not necessary anymore. Closing...

**Pull request closed without merging**